content/src/content/jcr_root/apps/core/wcm/components/form/text/v2/text/_cq_dialog/.content.xml

```diff
@@ -21,7 +21,8 @@
     jcr:title="Form Text Field"
     sling:resourceType="cq/gui/components/authoring/dialog"
     extraClientlibs="[core.wcm.components.form.text.v2.editor]"
-    helpPath="https://www.adobe.com/go/aem_cmp_form_text_v2">
+    helpPath="https://www.adobe.com/go/aem_cmp_form_text_v2"
+    trackingFeature="core-component:form:text:v2">
```

Comment:
does the `form` hierarchy provide any benefit in the data output? Or could we simply follow the same pattern for all components <namespace>:<component-name>:<version>. With the name of the form component prefixed with `form-` as we do with the BEM classes.

`core-components:form-text:v1`
`core-components:text:v2`
`core-components:contentfragment:v1`